frontend/public/components/modals/_modals.scss

```diff
@@ -26,6 +26,7 @@
   padding: 0;
   position: relative;
   -webkit-overflow-scrolling: touch;
+  overflow: scroll;
```

Comment:
While this does make the install modal scroll correctly, it may have unexpected behavior in other places where the `console-shared` `Modal` component is used.

Since the bug is happening because the markup structure is no longer matching what is in master, another way to fix this is to revert those changes. Here is a PR that does that and the scroll behavior is now isolated to only the install modal body content, if that is the path that the team wants to take with this fix. https://github.com/rhamilto/console/pull/55

Reply:
Great work, @sg00dwin.  I think your fix is the better option.